plugin-agent-admin-v2/src/components/AgentAdminView/AgentAdminView.jsx

```diff
@@ -12,86 +12,75 @@ import {
 
 import WorkerAttributes from '../WorkerAttributes/WorkerAttributes';
 
-const INITIAL_STATE = {
-  selectedWorker: undefined
-};
-
-class AgentAdminView extends React.Component {
+const AgentAdminView = () => {
+  const [selectedWorker, setSelectedWorker] = useState('');
+  const [sort, setSort] = useState({ name: "asc" })
+  const [teamFilterValue, setTeamFilterValue] = useState('');
+  const [skillsFilterValue, setSkillsFilterValue] = useState('');
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      selectedWorker: undefined,
-      sort: { name: "asc" },
-      filters: { team: "", skills: "" }
-    }
+  const workers = useSelector(
+    state => { return state['agent-admin']?.workerList?.workers || [] }
```

Comment:
You could remove the return statement here as well as the block body and rely on the implicit return.